src/mappings/serializeLikeHarvester.ts

```diff
@@ -76,6 +76,8 @@ export const serializeLikeHarvester = (
     return removeNullChars(item.toString());
   } else if (type === 'Ticker' || type === 'PolymeshPrimitivesTicker') {
     return serializeTicker(item);
+  } else if (rawType.startsWith('[u8;') && rawType.endsWith(']')) {
+    return item.toHex();
```

Comment:
Types such as IdentityId, CddId etc weren't properly serialized with latest changes, so had to add this in.